pixels-turbo/pixels-invoker-vhive/src/main/java/io/pixelsdb/pixels/invoker/vhive/utils/Utils.java

```diff
@@ -0,0 +1,435 @@
+package io.pixelsdb.pixels.invoker.vhive.utils;
+
+import com.alibaba.fastjson.JSON;
+import io.pixelsdb.pixels.executor.aggregation.FunctionType;
+import io.pixelsdb.pixels.executor.join.JoinType;
+import io.pixelsdb.pixels.planner.plan.physical.domain.*;
+import io.pixelsdb.pixels.planner.plan.physical.input.*;
+
+import java.util.*;
+import java.util.function.BiFunction;
+
+public class Utils
+{
+    public static AggregationInput genAggregationInput(StorageInfo storageInfo)
+    {
+        AggregationInput aggregationInput = new AggregationInput();
+        aggregationInput.setTransId(123456);
+        AggregatedTableInfo aggregatedTableInfo = new AggregatedTableInfo();
+        aggregatedTableInfo.setParallelism(8);
+        aggregatedTableInfo.setStorageInfo(storageInfo);
+        aggregatedTableInfo.setInputFiles(Arrays.asList(
```

Comment:
This file is full of hard-coded parameters. It should not exist in the main source folder. Move it to the test source folder if it is used for unit tests.